src/NServiceBus.RavenDB/Outbox/OutboxPersister.cs

```diff
@@ -109,16 +108,16 @@ public async Task SetAsDispatched(string messageId, ContextBag options, Cancella
                 if (useClusterWideTransactions)
                 {
                     var compareExchangeKey = $"rvn-atomic/{outboxRecordId}";
-                    var outboxRecord = await session.LoadAsync<OutboxRecord>(outboxRecordId, includes => includes.IncludeCompareExchangeValue(outboxRecordId), cancellationToken)
-                        .ConfigureAwait(false);
-                    var compareExchangeValue = await session.Advanced.ClusterTransaction.GetCompareExchangeValueAsync<CompareExchangeValue<string>>(compareExchangeKey, cancellationToken)
-                        .ConfigureAwait(false);
+                    var outboxRecordLazy = session.Advanced.Lazily.LoadAsync<OutboxRecord>(outboxRecordId, cancellationToken);
```

Comment:
Personally like to prevent multiple statements with nested properties but would not provide much more clarity here as it is super obvious.